packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -232,8 +235,19 @@ function Tab({
     defaultActionProps
   );
 
+  const { listeners, setNodeRef, transform, transition } = useSortable({
+    id: tabContentId,
+  });
+  const style = {
+    transform: cssDndKit.Transform.toString(transform),
+    transition,
+    cursor: 'grabbing !important',
```

Comment:
Doesn't look like this is being applied:

<img width="297" alt="image" src="https://user-images.githubusercontent.com/5036933/211823243-ae1e3371-6fc8-40fc-97ff-6951c205fab7.png">

Something overrides this `style` prop?

Reply:
Assigned styles directly on the dragging tab. Hopefully should be ok now.